src/Omnipay/Common/Helper.php

```diff
@@ -33,6 +34,26 @@ function ($match) {
     }
 
     /**
+     * Convert strings with underscores to be all lowercase before camelCase is preformed.
+     *
+     * @param  string $str The input string
+     * @return string The output string
+     */
+    protected static function convertToLowercase($str)
```

Comment:
You can update your branch without needing to create a new PR. The PR always looks at the head of the branch, so will be up-to-date with any changes you make (on that branch).